generative/maisi/maisi_train_controlnet_tutorial.ipynb

```diff
@@ -0,0 +1,528 @@
+{
```

Comment:
<p>isort gives the following imports</p><p> </p>

<pre class="ql-syntax" spellcheck="false">import copy
import json
import os
import subprocess


import nibabel as nib
import numpy as np
from monai.config import print_config
from monai.data import create_test_image_3d
from scripts.diff_model_setting import setup_logging


print_config()


logger = setup_logging("notebook")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1763/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Thanks, I reodered the import.